keras/layers/advanced_activations.py

```diff
@@ -73,7 +91,7 @@ def build(self, input_shape):
 
     def call(self, x, mask=None):
         pos = K.relu(x)
-        neg = self.alphas * (x - abs(x)) * 0.5
+        neg = K.pattern_broadcast(self.alphas, self.param_broadcast) * (x - abs(x)) * 0.5
```

Comment:
I would rather have a Theano-only branch here `if K.backend() == 'theano'` than add a dummy method to the TF backend.